Connection.php

```diff
@@ -555,3 +555,26 @@ public function close($code = self::CLOSE_NORMAL, $reason = null)
         return $connection->disconnect();
     }
 }
+
+/**
+ * Register socket wrappers
```

Comment:
Maybe, but the flow is not better with a static method... If we moved here we always need to think calling the static before start working...